.github/CODEOWNERS

```diff
@@ -1 +1,9 @@
 * @sfc-gh-turbaszek @sfc-gh-pjob @sfc-gh-jsikorski @sfc-gh-astus
+
+# Native Apps Owners
+src/snowcli/cli/nativeapp/ @sfc-gh-bgoel @sfc-gh-cgorrie @sfc-gh-bdufour @sfc-gh-melnacouzi
+tests/nativeapp/ @sfc-gh-bgoel @sfc-gh-cgorrie @sfc-gh-bdufour @sfc-gh-melnacouzi
+
+# Project Definition Owners
+src/snowcli/cli/project/ @sfc-gh-turbaszek @sfc-gh-pjob @sfc-gh-jsikorski @sfc-gh-astus @sfc-gh-bgoel @sfc-gh-cgorrie @sfc-gh-bdufour @sfc-gh-melnacouzi
```

Comment:
We could consider splitting the schema up into a folder with files and having Native Apps specifically own the native apps-relevant portions. Not blocking

Reply:
Agree on the suggestion. However, I think we can remove the snowcli team from here, the L1 makes us codeowners of everything